_includes/macros/category-icon.html

```diff
@@ -0,0 +1,41 @@
+
+{# ==========================================================================
+
+   category_icon()
+
+   ==========================================================================
+
+   Description:
+
+   Create an HTML icon when given:
+
+   category:           A document type category.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro render(category, additional_classes) %}
+    {%- if category|lower == 'info for consumers' -%}
```

Comment:
Ha, forgot I had already made this an issue https://github.com/cfpb/cfgov-refresh/issues/256 I think it's also in Jira on the BEWD board.